src/lib/utils/getAppIdAndApiKey.ts

```diff
@@ -1,5 +1,7 @@
 // typed as any, since it accepts the _real_ js clients, not the interface we otherwise expect
-export function getAppIdAndApiKey(searchClient: any) {
+export function getAppIdAndApiKey(
+  searchClient: any
```

Comment:
I looked into typing this, but as it accepts either v3 or v4, it would need some ts-ignore's like algoliasearch.ts and that doesn't seem worth it